From 95c9947e6c54ce2570c25b7724264981a76bdc2e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Techet?= <techet@gmail.com>
Date: Sat, 20 Nov 2021 18:18:18 +0100
Subject: [PATCH] Use dark-theme friendly colors for compiler messages

These colors seem to provide good legibility for both light and dark
themes and seem to be better defaults regardless of what theme users
use.
---
 data/filedefs/filetypes.common |  2 +-
 data/geany.css                 | 14 +++++++-------
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/data/filedefs/filetypes.common b/data/filedefs/filetypes.common
index a749bb2d..38a4e974 100644
--- a/data/filedefs/filetypes.common
+++ b/data/filedefs/filetypes.common
@@ -94,7 +94,7 @@ line_height=0;0;
 calltips=call_tips
 
 # error indicator color
-indicator_error=0xff0000
+indicator_error=0xff3030
 
 [settings]
 # which characters should be skipped when moving (or included when deleting) to word boundaries
diff --git a/data/geany.css b/data/geany.css
index ff1fe354..41a7aa00 100644
--- a/data/geany.css
+++ b/data/geany.css
@@ -28,27 +28,27 @@
 
 /* document status colors */
 #geany-document-status-changed {
-	color: #ff0000;
+	color: #ff3030;
 }
 #geany-document-status-disk-changed {
-	color: #ff7f00;
+	color: #ee8000;
 }
 #geany-document-status-readonly {
-	color: #007f00;
+	color: #309030;
 }
 
 /* compiler message colors */
 #geany-compiler-error {
-    color: #ff0000;
+    color: #ff3030;
 }
 #geany-compiler-context {
-    color: #7f0000;
+    color: #ee8000;
 }
 #geany-compiler-message {
-    color: #0000D0;
+    color: #006eff;
 }
 
 /* red "Terminal" label when terminal dirty */
 #geany-terminal-dirty {
-	color: #ff0000;
+	color: #ff3030;
 }
-- 
2.31.1

